TrainingProject/src/main/java/com/example/trainningspringproject/controller/MachineController.java

```diff
@@ -0,0 +1,62 @@
+package com.example.trainningspringproject.controller;
+
+import com.example.trainningspringproject.entity.Machine;
+import com.example.trainningspringproject.service.MachineService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class MachineController {
+
+    @Autowired
+    private MachineService service;
+
+    @GetMapping("/")
+    public String viewHomePage(Model model) {
+        List<Machine> listOfMachines = service.listAll();
+        model.addAttribute("listOfMachines", listOfMachines);
+        System.out.print(listOfMachines);
+        return "index";
+    }
+
+    @GetMapping("/tenant/{id}/machine/device")
+    public String viewUserHome(@PathVariable int id, Model model){
+        System.out.print(id);
```

Comment:
we use logs instead of printing to the console 
plea\se check it and add it to the project